tests/Aspire.Hosting.PostgreSQL.Tests/PostgresFunctionalTests.cs

```diff
@@ -229,6 +229,80 @@ public async Task WithDataShouldPersistStateBetweenUsages(bool useVolume)
         }
     }
 
+    [Fact]
+    [RequiresDocker]
+    public async Task VerifyWithInitBindMount()
+    {
+        // Creates a script that should be executed when the container is initialized.
+
+        var cts = new CancellationTokenSource(TimeSpan.FromMinutes(3));
+        var pipeline = new ResiliencePipelineBuilder()
+            .AddRetry(new() { MaxRetryAttempts = 10, Delay = TimeSpan.FromSeconds(1), ShouldHandle = new PredicateBuilder().Handle<NpgsqlException>() })
+            .Build();
+
+        var bindMountPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+
+        try
+        {
+            if (Directory.Exists(bindMountPath))
+            {
+                Directory.Delete(bindMountPath);
+            }
+
+            Directory.CreateDirectory(bindMountPath);
+
```

Comment:
```suggestion
        var bindMountPath = Directory.CreateTempSubdirectory();

        try
        {
```

This API is guaranteed to create a new, empty directory.